client/src/components/FilterMenu.js

```diff
@@ -45,40 +47,40 @@ class FilterMenu extends Component {
         this.props.changeLocation(latLng);
       })
       .catch(error => console.error('Error', error));
-  }
+  };
 
   toggleCategories = () => {
     this.setState(prevState => ({
       isShowCategories: !prevState.isShowCategories,
     }));
-  }
+  };
 
   toggleIndustries = () => {
     this.setState(prevState => ({
       isShowIndustries: !prevState.isShowIndustries,
     }));
-  }
+  };
 
   toggleSalary = () => {
     this.setState(prevState => ({
       isShowSalary: !prevState.isShowSalary,
     }));
-  }
+  };
 
-  onChangeSalary = (event) => {
+  onChangeSalary = event => {
```

Comment:
I think we should keep the parentheses around `event` for coding standards consistency, and it's easier to read.